### PR TITLE
Remove unused MACROs and Configurations for ALC5658 driver

### PR DESCRIPTION
**Pull request by btheosam**
ALC5658 codec driver don't use worker thread and hence no messages. This patch removes the MACROs and Configurations related to these.

### CODE REVIEW

File: os/include/tinyara/audio/alc5658.h [hunk 1]
```diff
@@ -120,10 +116,6 @@
 #error CONFIG_ALC5658_INFLIGHT must fit in a uint8_t
 #endif
 
-#ifndef CONFIG_ALC5658_MSG_PRIO
-#define CONFIG_ALC5658_MSG_PRIO          1
-#endif
-
 #ifndef CONFIG_ALC5658_BUFFER_SIZE
 #define CONFIG_ALC5658_BUFFER_SIZE       8192
```
Comment (jsdosa) [inline os/include/tinyara/audio/alc5658.h]: this value should change 8192 to 2048.
is that correct?

Comment (btheosam) [reply os/include/tinyara/audio/alc5658.h]: No, if no value is assigned, then better to keep the default value as 8k.

File: framework/src/tinyalsa/Make.defs [hunk 2]
```diff
@@ -15,8 +15,10 @@
  # language governing permissions and limitations under the License. 
  # 
  ########################################################################### 
-CSRCS += tinyalsa.c 
- 
+
+ifeq ($(CONFIG_AUDIO),y)
```
Comment (sunghan-chang) [inline framework/src/tinyalsa/Make.defs]: In Makefile of framework, that is already checked. DUPLICATED. REVERT IT.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Comment by sunghan-chang** [comment]
You need to resolve conflicts and to fill commit descrptions.

**Comment by seinfra** [comment]
Code Rule Check OK.

**Pull request closed without merging**